src/@next/components/organisms/ProductGallery/styles.ts

```diff
@@ -79,6 +79,7 @@ export const Preview = styled.div`
   max-height: 560px;
   overflow: hidden;
   img {
+    max-height: 100%;
```

Comment:
Is `100%` the max amount or the default? If it's default set it to `height: 100%`